### PR TITLE
Added 2024 Q3 report for Web3j

### PR DESCRIPTION
**Pull request by NickSneo**

### CODE REVIEW

**Review by petermetz — APPROVED** [review]

**Review by tkuhrt — APPROVED** [review]

**Review by swcurran — APPROVED** [review]

File: project-reports/2024/2024-Q2-Hyperledger-Web3j.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+layout: default
+title: 2024 Q2 Hyperledger Web3j
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q2-Hyperledger-Web3j.md]:
```suggestion
title: 2024 Q3 Hyperledger Web3j
```

File: project-reports/2024/2024-Q2-Hyperledger-Web3j.md
Comment (tkuhrt) [inline project-reports/2024/2024-Q2-Hyperledger-Web3j.md:1]: Need to rename this to reflect Q3 report

**Review by denyeart — APPROVED** [review]

File: project-reports/2024/2024-Q2-Hyperledger-Web3j.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+# Overall Activity in the Past Quarter
+
+- 100+ commits and 30 PRs merged in Web3j core.
+- commits and 34 PRs merged across all other Web3j repositories.
+- Fixed bugs, issues, and vulnerabilities, and updated Java and Kotlin versions.
+- Simplified and fixed the release process of Web3j after migrating to the Hyperledger GitHub organization.
+- Upgraded to Gradle 8.7 and updated all dependencies. [Check Announcement](https://medium.com/web3labs/boost-dev-efficiency-security-with-web3js-gradle-8-7-upgrade-d8435a9fb990).
+- Hyperledger Mentorship Program - Mentees started working on web3j-android. [Check project roadmap](https://wiki.hyperledger.org/display/INTERN/Project+Plan+-+Hyperledger+Web3j%3A+Enhancing+Android+Support+with+Updated+web3j-android+Integration).
+- Set up a fortnightly contributors call. A few people joined the call, but it is not very successful yet.
```
Comment (VRamakrishna) [inline project-reports/2024/2024-Q2-Hyperledger-Web3j.md]: Do the maintainers at least join this call regularly?

**Review by VRamakrishna — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

File: project-reports/2024/2024-Q2-Hyperledger-Web3j.md
Comment (NickSneo) [reply project-reports/2024/2024-Q2-Hyperledger-Web3j.md:1]: Updated to Q3

File: project-reports/2024/2024-Q2-Hyperledger-Web3j.md (hunk 2 above)
Comment (NickSneo) [reply project-reports/2024/2024-Q2-Hyperledger-Web3j.md]: Yes. Currently, @gtebrean and I are the only maintainers and regularly hold these meetings.

**Pull request merged**